.github/workflows/maintenance_bot.yaml

```diff
@@ -19,8 +19,13 @@ jobs:
           ! contains(github.event.pull_request.labels.*.name, 'status/WIP') &&
           ! contains(github.event.pull_request.title, 'WIP') &&
           ! github.event.pull_request.draft
-        run: |
-          curl --request PATCH \
-          --url ${{ github.event.pull_request.issue_url }} \
-          --header 'authorization: token ${{ secrets.GITHUB_TOKEN }}' \
-          --data-raw '{"milestone": '$MILESTONE_NUMBER'}'
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.issues.update({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              milestone: ${{ env.MILESTONE_NUMBER }},
+            });
```

Comment:
Oh, that's sweet. I wish I knew this earlier

Reply:
I found out by looking at the [`github-script` action](https://github.com/actions/github-script) you used for the labels verifier, which refers to https://octokit.github.io/rest.js/ . Just replace `octokit.` with `github.` and you have all the power of the GitHub REST API toolkit!